metabrainz/templates/index/projects.html

```diff
@@ -0,0 +1,134 @@
+{% extends 'base.html' %}
+
+{% block title %}Projects - MetaBrainz Foundation{% endblock %}
+
+{% block content %}
+<h1 class="page-title">The MetaBrainz Foundation Projects</h1>
+<div id="projects">
+  
+  <div class="media">
+    <div class="media-left media-middle">
+      <a href="https://musicbrainz.org/">
+        <img class="media-object" src="{{ url_for('static', filename='img/projects/MBnt.png') }}" alt="MusicBrainz logo">
+      </a>
+    </div>
+    <div class="media-body">
+      <h4 class="media-heading"><strong>MusicBrainz</strong></h4>
```

Comment:
I feel like it's superfluous to have `<strong>` within an `<h4>`. Just mark `h4`s as `font-weight: bold;` or whatever it is. Possibly `h4.media-heading`. Ask Roman or Leo.